packages/fuels-abigen-macro/Cargo.toml

```diff
@@ -14,12 +14,14 @@ proc-macro = true
 [dependencies]
 fuel-tx = "0.9"
 fuels-core = { version = "0.13.0", path = "../fuels-core" }
+log = "0.4"
```

Comment:
`fuel-core` is using `tracing` lib

Reply:
`tracing` is not much harder to setup: `tracing-subscriber` is `env-logger`-compatible too. Maybe we should consolidate?

Reply:
agree with using `tracing`

Reply:
Folks, although `tracing` has simple logging constructions, its main purpose is to do the tracing of spread events using `spans`. This is ideal for highly-concurrent systems, distributed systems, etc. because one can keep track of context and pass around spans representing the same event so that tools like grafana can plot metrics pertaining to each whole execution trace. I recommend reading it more [here](https://docs.rs/tracing/latest/tracing/).

This is very unnecessary for the SDK; it will add unnecessary complexity (the SDK already has enough complexities of its own 😂 ) for something that ought to be simple.

Reply:
I implemented it with both! There is currently “tracing” on the branch but we can revert it to "log" && "env_log".

Reply:
Nah, since you already did let's keep it. If it, somehow, becomes a pain later, we can yank it out 😄

Reply:
I just thought that using the same deps accross repos was nice, didn't think it was such a power tool haha